src/styles/card.css

```diff
@@ -0,0 +1,105 @@
+/*  CARD BLOCK:  */
+
+.card {
+  padding: 32px 0 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+}
+
+.card__info {
+  display: flex;
+  flex-direction: column;
+  margin: 0 16px;
+  --dark-color: #060b35;
+  --grey-color: #616070;
+}
+
+.card__info-logo {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__info-title {
+  color: var(--dark-color);
+  font-family: inherit;
+  font-weight: 500;
+  font-size: 12px;
+
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.card__info-code {
+  font-family: inherit;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--grey-color);
+  margin: 0 0 16px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  line-height: 14px;
+  margin-bottom: 24px;
+}
+
+.card__review-mark {
+  font-family: inherit;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-family: inherit;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--grey-color);
+}
+
+.card__price-amount {
+  font-family: inherit;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--dark-color);
+}
+
+.card__button {
+  height: 40px;
+
+  color: #fff;
+  background: #00acdc;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+
+  font-weight: 700;
+  font-size: 14px;
+  font-family: inherit;
+  line-height: 16px;
+  text-transform: uppercase;
+
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+
+  transition: all 0.5s ease-out;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated.

Reply:
Agree with @grafit-off